src/test/fuzz/script_bitcoin_consensus.cpp

```diff
@@ -28,4 +28,20 @@ FUZZ_TARGET(script_bitcoin_consensus)
     }
     (void)bitcoinconsensus_verify_script(random_bytes_1.data(), random_bytes_1.size(), random_bytes_2.data(), random_bytes_2.size(), n_in, flags, err_p);
     (void)bitcoinconsensus_verify_script_with_amount(random_bytes_1.data(), random_bytes_1.size(), money, random_bytes_2.data(), random_bytes_2.size(), n_in, flags, err_p);
+
+    if (n_in > 24386) return;
```

Comment:
I think we should still call `bitcoinconsensus_verify_script_with_spent_outputs` when `n_in` is too large. Just only allocate when it's not.

(I initially pointed this out in https://github.com/bitcoin/bitcoin/pull/28539#issuecomment-1741775909 but my email client messed up my message, sorry about that.)

Reply:
Done, thanks!